packages/perseus/src/widgets/numeric-input.tsx

```diff
@@ -481,7 +473,7 @@ export const unionAnswerForms: (
 
 type RenderProps = {
     answerForms: ReadonlyArray<{
-        simplify: "required" | "correct" | "enforced" | null | undefined;
+        simplify: Simplify | null | undefined;
```

Comment:
These are a slightly different set of options than defined in the `Simplify` type. I suspect that was an error when we typed things originally, but wanted to point it out. 

https://github.com/Khan/perseus/pull/504/files?diff=unified&w=1#diff-5737e5c75b244a64356894bb42c711539f1589cfbd9dcbbd4b22a3db8773d524R719